core/mpc/mpc.ts

```
@@ -26,7 +26,7 @@ export class MPC {
     localPartyId: string,
     keygenCommittee: string[],
     oldKeygenCommittee: string[],
-    hexEncryptionKey: string
+    hexEncryptionKey: string,
```

ESLint

[error] Delete `,`

(prettier/prettier)